v2/blockstore/readwrite_test.go

```diff
@@ -530,7 +534,7 @@ func TestReadWriteResumptionFromFileWithDifferentCarV1PaddingIsError(t *testing.
 	subject, err := blockstore.OpenReadWrite(
 		path,
 		WantRoots,
-		blockstore.WithCarV1Padding(1413))
+		carv2.UseCarV1Padding(1413))
```

Comment:
I thought about this for a little bit. I dislike having every option start with `With` because it's a bit redundant, and sometimes a bit confusing. For example, an option called WithNullPadding could be understood as either "allow null padding when parsing", or "use N null padding when writing". On the other hand, UseNullPadding is clearly "write padding", and AllowNullPadding is "allow parsing null padding".

Reply:
I do realise we're using With for the Header type, but I think there it's fine - it's a method that returns a modified copy of the header, so the "with" there carries meaning - it's like "a copy but *with* X set".